go.mod

```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/cloudflare/ipvs v0.8.0
+	github.com/coreos/go-iptables v0.4.5
```

Comment:
why not use https://github.com/coreos/go-iptables/releases/tag/v0.6.0 ?

Reply:
thx; updated